### PR TITLE
Font Updates

### PR DESCRIPTION
**Pull request by rjspiker**
Preview: https://nomad-fonts.netlify.com

This PR accomplishes the following:
- Add newly approved fonts (Gilmer, Metro Sans, Deja Vu) and remove deprecated fonts
- Global typography updates (font-size, line-height, etc), adding `.g-type` classes shared across sites
- Color updates to match color scheme with other Hashicorp sites
- Button style overhaul

### CODE REVIEW

**Comment by kylemac** [comment]
@rjspiker do we need to do anything to support the homepage eyebrow or will that work when it's on production? https://tppr.me/JBQxw

File: website/source/index.html.erb [hunk 1]
```diff
@@ -11,7 +11,7 @@ description: |-
       <div class="col-md-offset-2 col-md-8">
         <%= inline_svg "logo-hashicorp.svg", height: 120, class: "logo" %>
 
-        <h1>Deploy and Manage Any Containerized, Legacy, or Batch Application</h1>
+        <h1 class="g-type-display-1">Deploy and Manage Any Containerized, Legacy, or Batch Application</h1>
```
Comment (kylemac) [inline website/source/index.html.erb]:
```suggestion
        <h2 class="g-type-display-3">Deploy and Manage Any Containerized, Legacy, or Batch Application</h1>
```

- for semantics, i think we should reduce this to an h2 as its a subheading
- for styling purposes, i think we should use `g-type-display-3`. the current version reads as way too big imo. curious to see what kristopher thinks...

**Review by kylemac — CHANGES_REQUESTED** [review]
1 question and 1 suggestion left. moving on to the next project...

**Comment by rjspiker** [comment]
> @rjspiker do we need to do anything to support the homepage eyebrow or will that work when it's on production? https://tppr.me/JBQxw

We just need to make sure it has the proper `g-type` classes on it.  I'll pull the latest to make sure it's up to date and add those classes.

File: website/source/index.html.erb (hunk 1 above)
Comment (rjspiker) [reply website/source/index.html.erb]: I can run this by Kristopher and see what he thinks.

My concern about changing this to an `h2` is around accessibility (https://dequeuniversity.com/rules/axe/3.3/page-has-heading-one?application=AxeChrome).  Thoughts?

Comment (kylemac) [reply website/source/index.html.erb]: ah i see. so semantically, we'd want the h1 to be "Nomad" here and this subheader to be an h2 - but changing that across sites should just be out of scope for this PR. I'm fine with keeping it h1 for now, knowing we'll be doing a more thorough a11y review in the near future. 

i do want to see us change the `g-type-display-3` though ;)

**Review by zchsh — APPROVED** [review]
✅ Looks really good, poked around the preview and didn't see anything that looked off.

Same note as on Consul applies about Typekit stuff and the "HashiCorp Suite" part, again though definitely seems like something we'd need to address on the design side, and something we can defer, so marking as approved 👍

**Comment by rjspiker** [comment]
Thanks for the reviews.  Going to open a PR upstream.

**Pull request merged**